src/main/java/org/jitsi/jicofo/MeetExtensionsHandler.java

```diff
@@ -240,8 +240,10 @@ private IQ handleRayoIQ(RayoIqProvider.DialIq dialIq, int retryCount,
         }
 
 
-        List<String> bridgeRegions = conference.getBridges()
-            .stream().map(b -> b.getRegion()).collect(Collectors.toList());
+        List<String> bridgeRegions = conference.getBridges().stream()
+            .map(b -> b.getRegion())
+            .filter(Objects::nonNull)
+            .collect(Collectors.toList());
```

Comment:
Why not `toSet`? We don't care about the order, and we don't want duplicates, right?